api/app.py

```diff
@@ -38,19 +34,48 @@ def get_algorithms():
 @app.route('/analyze', methods=['POST'])
 @cross_origin(origin=['*'],supports_credentials=True)
 def analyze():
-    # export platform info to prometheus
-    response = requests.post(qujata_platform_exporter_target + "/export-platform-info")
-
     global test_is_running
```

Comment:
how come we define this global inside this API? I can also see this var on the top of the page, but not a global var.
Won't
**global** test_is_running = False
on the top of the page work?